src/ngraph/runtime/gpu/CMakeLists.txt

```diff
@@ -41,6 +41,9 @@ set(SRC
     pass/gpu_layout.cpp
     pass/tensor_memory_reservation.cpp
     gpu_kernel_args.cpp
+    pass/gpu_rnn_fusion.cpp
+    ../gpu/op/lstm.cpp
```

Comment:
replace the `../gpu` with something more elegant like ${CMAKE_CURRENT_SOURCE_DIR}